assets/grafana/celeborn-dashboard.json

```diff
@@ -440,21 +426,21 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "editorMode": "code",
-              "expr": "metrics_IsActiveMaster_Value",
+              "expr": "metrics_ActiveShuffleSize_Value",
```

Comment:
ActiveShuffleSize and ActiveShuffleFileCount will display the values of all instances in the overall row without role-related filtering. This is repeated with this two panels in the worker row.
IMO in the overall row, we can only display the sum values of the two metrics.(like master exposed in this pr) The worker dimension may be a bit too much here.